### PR TITLE
90% Author accessibility content

### PR DESCRIPTION
**Pull request by danielmatthew**
An introduction and guidelines to the principles we should be applying within our products.

### CODE REVIEW

File: topics/accessibility.md [hunk 1]
```diff
@@ -5,11 +5,107 @@ body_class: accessibility
...
+
+#### Links
+Avoid using generic links such as "Read More". Where unavoidable, use `aria-label` to provide a more verbose description of the link contents, or `aria-labelledby` to reference a more descriptive element. While the `title` attribute might seem logical, it is ignored by screen readers.
+
+{% highlight html %}
+<h1 id="brown-fox-title">The Brown Fox</h1>
+
+<!-- Screen reader: "Link: Article about the brown fox" -->
+<a href="#" aria-label="Article about the brown fox">Read more</a>
+<!-- Screen reader an "Link; The Brown Fox" -->
```
Comment (russellhill) [inline topics/accessibility.md]: what's the comment here trying to say?

File: topics/accessibility.md [hunk 2]
```diff
@@ -5,11 +5,107 @@ body_class: accessibility
...
+<a href="#" aria-labelledby="brown-fox-title">Read more</a>
+{% endhighlight %}
+
+
+#### Images
+The `alt` attribute on images is required in an effort to conform to accessibility guidelines. It should provide a concise summary of the image contents. If this is not possible, leave it empty: an empty attribute is better than a missing one as the assistive technology will ignore it. A more verbose description can be added using with `aria-describedby` - this is useful if there is an associated caption.
+
+#### Tabs
+Elements that represent tab controls should assume `role="tab"`, and be linked to the relevant content using the `arial-controls` attribute. The content itself implements the `tabpanel` role and `aria-labelledby`.
```
Comment (russellhill) [inline topics/accessibility.md]: should that be ```arial-controls``` or ```aria-controls```?

**Comment by russellhill** [comment]
:+1:

File: topics/accessibility.md (hunk 2 above)
Comment (danielmatthew) [inline topics/accessibility.md]: Fonts on the brain!

**Pull request merged**